connector/rest/src/main/java/it/pagopa/selfcare/dashboard/connector/rest/model/mapper/InstitutionV2Mapper.java

```diff
@@ -0,0 +1,30 @@
+package it.pagopa.selfcare.dashboard.connector.rest.model.mapper;
+
+import it.pagopa.selfcare.dashboard.connector.model.institution.InstitutionInfo;
+import it.pagopa.selfcare.dashboard.connector.model.institution.RelationshipState;
+import it.pagopa.selfcare.user.generated.openapi.v1.dto.InstitutionProducts;
+import it.pagopa.selfcare.user.generated.openapi.v1.dto.OnboardedProductResponse;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+
+@Mapper(componentModel = "spring")
+public interface InstitutionV2Mapper {
```

Comment:
you could have only one InstitutionMapper without version that refer to connector model....